src/main/java/org/prebid/server/bidder/model/BidderErrorInfo.java

```diff
@@ -0,0 +1,14 @@
+package org.prebid.server.bidder.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+/**
+ * Stores error information: error code and message
+ */
+@Builder
```

Comment:
Per convention for POJOs having up to 4 members (inclusive) static factory method is preferred over builder